lib/zendesk_apps_tools/bump.rb

```diff
@@ -0,0 +1,29 @@
+require 'thor'
+require 'json'
+
+require 'zendesk_apps_tools/manifest_handler'
+
+module ZendeskAppsTools
+  class Bump < Thor
+    include Thor::Actions
+    prepend ManifestHandler
```

Comment:
Have not seen this before, I found it rather confusing, but seems to make sense now you've explained it to me.